### PR TITLE
8.0 web advancd search wildcard

### PR DESCRIPTION
**Pull request by lfreeke**
Port of 7.0 web advancd search wildcard

### CODE REVIEW

**Comment by hbrunn** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Can you please keep commit history for preserving the attribution? You can follow a similar mechanism than this one: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0#if-the-module-doesnt-exist-in-the-90-branch

**Comment by lfreeke** [comment]
@pedrobaeza Thanks, Done.

**Comment by elicoidal** [comment]
:+1:

File: web_advanced_search_wildcard/README.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+============================
+Web advanced search wildcard
+============================
+
+Allows =ilike operator to advanced search option.
+
+Use % as a placeholder.
```
Comment (pedrobaeza) [inline web_advanced_search_wildcard/README.rst:13]: Put this on Usage section

File: web_advanced_search_wildcard/README.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Example: "Zip matches 1%" gives all zip starting with 1
```
Comment (pedrobaeza) [inline web_advanced_search_wildcard/README.rst]: It's better understood as `"Zip" - "matches" - "1%"`

File: web_advanced_search_wildcard/README.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+============================
+Web advanced search wildcard
+============================
+
+Allows =ilike operator to advanced search option.
```
Comment (pedrobaeza) [inline web_advanced_search_wildcard/README.rst]: s/to/in

Comment (pedrobaeza) [inline web_advanced_search_wildcard/README.rst]: Add that the operator is called "Matches" in the UI.

File: web_advanced_search_wildcard/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2014 initOS GmbH & Co. KG (<http://www.initos.com>).
+# © 2016 Therp BV <http://therp.nl>.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name":     "Webmodule add wildcard operators for advanced search",
+    "summary":  "Simular search in searchbar",
```
Comment (pedrobaeza) [inline web_advanced_search_wildcard/__openerp__.py]: s/Simular search in searchbar/Pattern search on advanced search

Comment (pedrobaeza) [inline web_advanced_search_wildcard/__openerp__.py]: Or maybe this is better as name

File: web_advanced_search_wildcard/README.rst [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+Use % as a placeholder.
+Example: "Zip matches 1%" gives all zip starting with 1
+Also allows insensitive exact search.
+Example "Name matches john" will find "John" and "john" but not "Johnson".
+
```
Comment (elicoidal) [inline web_advanced_search_wildcard/README.rst:23]: Adding a screenshot would be nice

File: web_advanced_search_wildcard/README.rst [hunk 6]
```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+============================
+Web advanced search wildcard
```
Comment (elicoidal) [inline web_advanced_search_wildcard/README.rst]: Wildcard in advanced search

File: web_advanced_search_wildcard/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2014 initOS GmbH & Co. KG (<http://www.initos.com>).
+# © 2016 Therp BV <http://therp.nl>.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name":     "Webmodule add wildcard operators for advanced search",
```
Comment (elicoidal) [inline web_advanced_search_wildcard/__openerp__.py]: Wildcard in advanced search

File: web_advanced_search_wildcard/__openerp__.py (hunk 4 above)
Comment (elicoidal) [inline web_advanced_search_wildcard/__openerp__.py]: Summary: Webmodule to add wildcard operators in advanced search field

File: web_advanced_search_wildcard/__openerp__.py [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+    "summary":  "Simular search in searchbar",
+    "version":  "8.0.1.0.0",
+    "category": "Uncategorized",
```
Comment (elicoidal) [inline web_advanced_search_wildcard/__openerp__.py]: web?

**Comment by lfreeke** [comment]
@pedrobaeza @elicoidal Thanks for the feedback. 
I applied the changes you both requested

**Comment by hbrunn** [comment]
@lfreeke is covered by Therp's company CLA, I informed the board about this

**Pull request merged**